src/platform/stm32wl/LittleFS.cpp

```diff
@@ -23,41 +23,46 @@
  */
 
 #include "LittleFS.h"
+#include "stm32wlxx_hal_flash.h"
 
 /**********************************************************************************************************************
  * Macro definitions
  **********************************************************************************************************************/
 /** This macro is used to suppress compiler messages about a parameter not being used in a function. */
-#define PARAMETER_NOT_USED(p) (void)((p))
+#define LFS_UNUSED(p) (void)((p))
 
-#define STM32WL_SECTOR_SIZE 0x800 /* 2K */
-#define STM32WL_SECTOR_COUNT 14
+#define STM32WL_PAGE_SIZE  (FLASH_PAGE_SIZE)
+#define STM32WL_PAGE_COUNT (FLASH_PAGE_NB)
+#define STM32WL_FLASH_BASE (FLASH_BASE)
 
-#define LFS_FLASH_TOTAL_SIZE (STM32WL_SECTOR_COUNT * STM32WL_SECTOR_SIZE)
-#define LFS_BLOCK_SIZE 128
+/* FLASH_SIZE from stm32wle5xx.h will read the actual FLASH size from the chip */
+/* use the last 1/4 of the FLASH */
```

Comment:
i'd rather stick to the 28kb proposed. In line with the nRF52840 flash space, which is plenty on there. Using 1/4th of the flash would waste 36kb

Reply:
See https://github.com/dchokola/meshtastic-firmware/commit/f2bff947bb97b1668b277ea20bfb19cad0be41aa